content/hardware/litebeamac.md

```diff
@@ -15,7 +15,10 @@ Gen2 comes with a more sturdy mount (though less range) and also a management 2.
 
 LiteBeams are very directional so use the built-in alignment tool to get the strongest signal. We like to get better than -65db. Very close to the supernodes you can get -45db.
 
-The default IP is https://192.168.1.20/ with name:ubnt pwd:ubnt
+The default IP is https://192.168.1.20/ with:
+
+* name:ubnt
+* pwd:ubnt
```

Comment:
formatted with bulletlist to be similar to other username / password formatting on other pages